package.json

```diff
@@ -8,55 +8,27 @@
   },
   "homepage": "https://cobalt-ui.pages.dev",
   "scripts": {
-    "prebuild": "del-cli \"**/dist\"",
-    "build": "run-s -s build:utils build:core build:packages",
-    "build:core": "cd packages/core && pnpm run build",
-    "build:examples": "run-p -s build:examples:*",
-    "build:examples:adobe": "cd examples/adobe && pnpm run build:tokens",
-    "build:examples:apple": "cd examples/apple && pnpm run build:tokens",
-    "build:examples:github": "cd examples/github && pnpm run build:tokens",
-    "build:examples:ibm": "cd examples/ibm && pnpm run build:tokens",
-    "build:examples:salesforce": "cd examples/salesforce && pnpm run build:tokens",
-    "build:examples:shopify": "cd examples/shopify && pnpm run build:tokens",
-    "build:utils": "cd packages/utils && pnpm run build",
-    "build:packages": "run-p -s build-packages:*",
-    "build-packages:cli": "cd packages/cli && pnpm run build",
-    "build-packages:plugin-css": "cd packages/plugin-css && pnpm run build && cd ../plugin-sass && pnpm run build && cd ../plugin-tailwind && pnpm run build",
-    "build-packages:plugin-js": "cd packages/plugin-js && pnpm run build",
-    "cf:build": "pnpm run build && cd docs && pnpm run build",
+    "build": "pnpm run -r --if-present --filter \"@cobalt-ui/*\" \"build\"",
+    "build:docs": "pnpm run -r --if-present \"build\"",
+    "build:examples": "pnpm run -r --parallel --if-present build:tokens",
     "changeset": "changeset",
-    "dev": "run-p -s dev:*",
-    "dev:core": "cd packages/core && pnpm run dev",
-    "dev:cli": "cd packages/cli && pnpm run dev",
-    "dev:utils": "cd packages/utils && pnpm run dev",
-    "dev:plugin-css": "cd packages/plugin-css && pnpm run dev",
-    "dev:plugin-js": "cd packages/plugin-js && pnpm run dev",
-    "dev:plugin-sass": "cd packages/plugin-sass && pnpm run dev",
-    "dev:plugin-tw": "cd packages/plugin-tailwind && pnpm run dev",
+    "dev": "pnpm run -r --parallel --if-present --filter \"@cobalt-ui/*\" dev",
     "lint": "eslint \"packages/**/*.{ts,js,cjs,mjs}\"",
-    "test": "run-p test:*",
-    "test:cli": "cd packages/cli && pnpm run test",
-    "test:core": "cd packages/core && pnpm run test",
-    "test:plugin-css": "cd packages/plugin-css && pnpm run test",
-    "test:plugin-js": "cd packages/plugin-js && pnpm run test",
-    "test:plugin-sass": "cd packages/plugin-sass && pnpm run test",
-    "test:plugin-tw": "cd packages/plugin-tailwind && pnpm run test",
-    "test:utils": "cd packages/utils && pnpm run test",
+    "test": "pnpm run -r --if-present --aggregate-output --parallel test",
```

Comment:
Note to literally no one but myself: `--parallel` on the top level is a good idea. Nested `--parallel`s is a horrible idea (i.e. don’t use within any packages).